spec/system/work_spec.rb

```diff
@@ -159,6 +172,42 @@
       expect(draft_work.resource.creators.last.given_name).to eq(creator.given_name)
       expect(draft_work.resource.creators.last.family_name).to eq(creator.family_name)
     end
+
+    it "allows users to modify the order of the contributors", js: true do
+      # Make the screen larger so the save button is alway on screen.  This avoids random `Element is not clickable` errors
+      page.driver.browser.manage.window.resize_to(2000, 2000)
+      sign_in user
+      visit edit_work_path(draft_work)
+      click_on "Additional Metadata"
+      fill_in "contributor_given_name_1", with: "Robert"
+      fill_in "contributor_family_name_1", with: "Smith"
+      click_on "Add Another Contributor"
+      fill_in "contributor_given_name_2", with: "Simon"
+      fill_in "contributor_family_name_2", with: "Gallup"
+
+      contributor_text = page.all("tr")[1..2].map { |each| each.all("input").map(&:value) }.flatten.join(" ").strip
+      expect(contributor_text).to eq("Robert Smith  Simon Gallup")
+
+      # drag the first contributor to the second contributor
+      source = page.all(".bi-arrow-down-up")[0].native
+      target = page.all(".bi-arrow-down-up")[1].native
+      builder = page.driver.browser.action
+      builder.drag_and_drop(source, target).perform
+      contributor_text_after = page.all("tr")[1..2].map { |each| each.all("input").map(&:value) }.flatten.join(" ").strip
+
+      # This is really strange, but my local machine likes to drag from bottom to top and CircleCI likes to drag
```

Comment:
@hectorcorrea did you see this same behavior, or did you just copy paste? I'm happy for the comment, and just curious...